src/cim/components/Agent/AgentsSelectionTable.tsx

```diff
@@ -111,13 +122,21 @@ const AgentsSelectionTable: React.FC<AgentsSelectionTableProps> = ({
 
   const paginationProps = usePagination(hosts.length);
 
+  const actionChecks = React.useMemo(
+    () => ({
+      canEditDisks: (host: Host) =>
+        canEditDisksUtil(cluster ? cluster.status : undefined, host.status),
+    }),
+    [cluster],
+  );
+
```

Comment:
canEditDisks can be as simple as ` (host) => true ` because the disks can be editable in AgentsSelectionTable